README.md

```diff
@@ -102,16 +137,17 @@ Neos:
             uriSegment: 'dk'
             options:
               deeplLanguage: 'da'
-              
-          #    
+              translationStrategy: 'once'
+
+          #
           # The bavarian language is not supported by DeepL and is disabled
-          # 
+          #
           'de_bar':
             label: 'Bayrisch'
             values: ['de_bar','de']
             uriSegment: 'de_bar'
             options:
-              deeplLanguage: ~
+              translationStrategy: 'sync'
```

Comment:
This should be `none` as deepl does not support bavarian yet

Reply:
Will correct that